yodeploy/test_integration/test_server.py

```diff
@@ -19,9 +18,12 @@ def setUp(self):
         self.username = self.flask_app.config.server.username
         self.password = self.flask_app.config.server.password
 
+        auth_header_val = '%s:%s' % (self.username, self.password)
+        auth_header_val = auth_header_val.encode()
+        auth_header_val = b'Basic %s' % base64.b64encode(auth_header_val)
```

Comment:
The codecs module gives us some neat tricks, like: `auth_header_val.encode('base64')`